mrp_project/__openerp__.py

```diff
@@ -6,10 +6,11 @@
 {
     "name": "MRP Project Link",
     "summary": "Link production with projects",
-    "version": "8.0.1.1.0",
+    "version": "8.0.1.1.1",
     "depends": [
         "mrp_analytic",
         "project",
+        "project_timesheet",
```

Comment:
OK, go on with it. You're not going to find any installation that enables real costs without it, but please change version to 8.0.1.2.0